### ISSUE TITLE
Pod Not Install

### ISSUE THREAD

**Issue by valoroustech** [CocoaPods/CocoaPods#9698]
pod install
Analyzing dependencies

――― MARKDOWN TEMPLATE ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――

### Command

```
/Applications/CocoaPods.app/Contents/Resources/bundle/bin/pod install
```

### Report

* What did you do?

* What did you expect to happen?

* What happened instead?

### Stack

```
   CocoaPods : 1.5.2
        Ruby : ruby 2.2.6p396 (2016-11-15 revision 56800) [x86_64-darwin15]
    RubyGems : 2.6.8
        Host : Mac OS X 10.14.4 (18E226)
       Xcode : 11.3 (11C29)
         Git : git version 2.6.2
Ruby lib dir : /Applications/CocoaPods.app/Contents/Resources/bundle/lib
Repositories : master - https://github.com/CocoaPods/Specs.git @ 16f7158eb43b395cf2e77f2cb0a6141f6613806f
```

### Plugins

```
cocoapods-check           : 1.0.0
cocoapods-deintegrate     : 1.0.2
cocoapods-plugins         : 1.0.0
cocoapods-plugins-install : 0.0.1
cocoapods-search          : 1.0.0
cocoapods-stats           : 1.0.0
cocoapods-trunk           : 1.3.0
cocoapods-try             : 1.1.0
```

### Podfile

```ruby
# Uncomment the next line to define a global platform for your project
 platform :ios, '9.0'

target 'FundSpace' do
  # Comment the next line if you're not using Swift and don't want to use dynamic frameworks
  use_frameworks!

  # Pods for FundSpace
  #pod 'SideMenu'
 #pod 'PageMenu'
 pod 'GoogleSignIn'
 pod 'FacebookLogin'
 pod 'Alamofire'
 pod 'JGProgressHUD'
 pod 'SDWebImage'
 #pod 'SkeletonView'
 pod 'ImageSlideshow', '~> 1.8'
 pod 'ImageSlideshow/Alamofire'
 pod 'AlamofireImage'
 pod 'IQKeyboardManagerSwift'
 pod 'SkyFloatingLabelTextField', '~> 3.0'
 pod 'SwiftyJSON', '~> 4.0'
 pod 'SVGKit', :git => 'https://github.com/SVGKit/SVGKit.git', :branch => '2.x'
 pod 'ObjectMapper', '~> 3.4'
 pod 'AlamofireObjectMapper', '~> 5.2'
 pod 'DatePickerDialog'
 pod 'McPicker'
 pod 'ReachabilitySwift'
 pod 'Firebase/Core'
 pod 'GoogleSignIn'
 pod 'Firebase/Database'
 pod 'RealmSwift'
 pod 'Firebase/Analytics'
 pod 'AppAuth','~> 1.2.0'
 pod 'Firebase/Auth'
 pod 'Firebase/Storage'
 pod 'Firebase/Messaging'
 pod 'Firebase/RemoteConfig'
 pod 'Charts'
 
 pod 'FirebaseInstanceID', "2.0.0"
  target 'FundSpaceTests' do
    inherit! :search_paths
    # Pods for testing
  end

  target 'FundSpaceUITests' do
    inherit! :search_paths
    # Pods for testing
  end

end
```

### Error

```
RuntimeError - [Xcodeproj] Unknown object version.
/Applications/CocoaPods.app/Contents/Resources/bundle/lib/ruby/gems/2.2.0/gems/xcodeproj-1.5.8/lib/xcodeproj/project.rb:218:in `initialize_from_file'
/Applications/CocoaPods.app/Contents/Resources/bundle/lib/ruby/gems/2.2.0/gems/xcodeproj-1.5.8/lib/xcodeproj/project.rb:103:in `open'
/Applications/CocoaPods.app/Contents/Resources/bundle/lib/ruby/gems/2.2.0/gems/cocoapods-1.5.2/lib/cocoapods/installer/analyzer.rb:918:in `block (2 levels) in inspect_targets_to_integrate'
/Applications/CocoaPods.app/Contents/Resources/bundle/lib/ruby/gems/2.2.0/gems/cocoapods-1.5.2/lib/cocoapods/installer/analyzer.rb:917:in `each'
/Applications/CocoaPods.app/Contents/Resources/bundle/lib/ruby/gems/2.2.0/gems/cocoapods-1.5.2/lib/cocoapods/installer/analyzer.rb:917:in `block in inspect_targets_to_integrate'
/Applications/CocoaPods.app/Contents/Resources/bundle/lib/ruby/gems/2.2.0/gems/cocoapods-1.5.2/lib/cocoapods/user_interface.rb:64:in `section'
/Applications/CocoaPods.app/Contents/Resources/bundle/lib/ruby/gems/2.2.0/gems/cocoapods-1.5.2/lib/cocoapods/installer/analyzer.rb:912:in `inspect_targets_to_integrate'
/Applications/CocoaPods.app/Contents/Resources/bundle/lib/ruby/gems/2.2.0/gems/cocoapods-1.5.2/lib/cocoapods/installer/analyzer.rb:78:in `analyze'
/Applications/CocoaPods.app/Contents/Resources/bundle/lib/ruby/gems/2.2.0/gems/cocoapods-1.5.2/lib/cocoapods/installer.rb:243:in `analyze'
/Applications/CocoaPods.app/Contents/Resources/bundle/lib/ruby/gems/2.2.0/gems/cocoapods-1.5.2/lib/cocoapods/installer.rb:154:in `block in resolve_dependencies'
/Applications/CocoaPods.app/Contents/Resources/bundle/lib/ruby/gems/2.2.0/gems/cocoapods-1.5.2/lib/cocoapods/user_interface.rb:64:in `section'
/Applications/CocoaPods.app/Contents/Resources/bundle/lib/ruby/gems/2.2.0/gems/cocoapods-1.5.2/lib/cocoapods/installer.rb:153:in `resolve_dependencies'
/Applications/CocoaPods.app/Contents/Resources/bundle/lib/ruby/gems/2.2.0/gems/cocoapods-1.5.2/lib/cocoapods/installer.rb:116:in `install!'
/Applications/CocoaPods.app/Contents/Resources/bundle/lib/ruby/gems/2.2.0/gems/cocoapods-1.5.2/lib/cocoapods/command/install.rb:41:in `run'
/Applications/CocoaPods.app/Contents/Resources/bundle/lib/ruby/gems/2.2.0/gems/claide-1.0.2/lib/claide/command.rb:334:in `run'
/Applications/CocoaPods.app/Contents/Resources/bundle/lib/ruby/gems/2.2.0/gems/cocoapods-1.5.2/lib/cocoapods/command.rb:52:in `run'
/Applications/CocoaPods.app/Contents/Resources/bundle/lib/ruby/gems/2.2.0/gems/cocoapods-1.5.2/bin/pod:55:in `<top (required)>'
/Applications/CocoaPods.app/Contents/Resources/bundle/bin/pod:22:in `load'
/Applications/CocoaPods.app/Contents/Resources/bundle/bin/pod:22:in `<main>'
```

――― TEMPLATE END ――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――

[!] Oh no, an error occurred.

Search for existing GitHub issues similar to yours:
https://github.com/CocoaPods/CocoaPods/search?q=%5BXcodeproj%5D+Unknown+object+version.&type=Issues

If none exists, create a ticket, with the template displayed above, on:
https://github.com/CocoaPods/CocoaPods/issues/new

Be sure to first read the contributing guide for details on how to properly submit a ticket:
https://github.com/CocoaPods/CocoaPods/blob/master/CONTRIBUTING.md

Don't forget to anonymize any private data!

Looking for related issues on cocoapods/cocoapods...
 - Pod init gives error message
   https://github.com/CocoaPods/CocoaPods/issues/9666 [closed] [1 comment]
   a week ago

 -        Xcode : 11.4 (11E146)
   https://github.com/CocoaPods/CocoaPods/issues/9662 [closed] [6 comments]
   2 days ago

 - Xcodeproj doesn't know about the following attributes {"inputFileListPaths"=>[], "outputFileListPaths"=>[]} for the 'PBXShellScriptBuildPhase' isa
   https://github.com/CocoaPods/CocoaPods/issues/7835 [closed] [54 comments]
   a week ago

and 50 more at:
https://github.com/cocoapods/cocoapods/search?q=%5BXcodeproj%5D%20Unknown%20object%20version.&type=Issues&utf8=✓

[!] [!] Xcodeproj doesn't know about the following attributes {"inputFileListPaths"=>[], "outputFileListPaths"=>[]} for the 'PBXShellScriptBuildPhase' isa.
If this attribute was generated by Xcode please file an issue: https://github.com/CocoaPods/Xcodeproj/issues/new

[!] [!] Xcodeproj doesn't know about the following attributes {"inputFileListPaths"=>["${PODS_ROOT}/Target Support Files/Pods-FundSpace/Pods-FundSpace-frameworks-${CONFIGURATION}-input-files.xcfilelist"], "outputFileListPaths"=>["${PODS_ROOT}/Target Support Files/Pods-FundSpace/Pods-FundSpace-frameworks-${CONFIGURATION}-output-files.xcfilelist"]} for the 'PBXShellScriptBuildPhase' isa.
If this attribute was generated by Xcode please file an issue: https://github.com/CocoaPods/Xcodeproj/issues/new

[!] [!] Xcodeproj doesn't know about the following attributes {"inputFileListPaths"=>["${PODS_ROOT}/Target Support Files/Pods-FundSpace/Pods-FundSpace-resources-${CONFIGURATION}-input-files.xcfilelist"], "outputFileListPaths"=>["${PODS_ROOT}/Target Support Files/Pods-FundSpace/Pods-FundSpace-resources-${CONFIGURATION}-output-files.xcfilelist"]} for the 'PBXShellScriptBuildPhase' isa.
If this attribute was generated by Xcode please file an issue: https://github.com/CocoaPods/Xcodeproj/issues/new

[!] [!] Xcodeproj doesn't know about the following attributes {"inputFileListPaths"=>[], "outputFileListPaths"=>[]} for the 'PBXShellScriptBuildPhase' isa.
If this attribute was generated by Xcode please file an issue: https://github.com/CocoaPods/Xcodeproj/issues/new

[!] [!] Xcodeproj doesn't know about the following attributes {"inputFileListPaths"=>[], "outputFileListPaths"=>[]} for the 'PBXShellScriptBuildPhase' isa.
If this attribute was generated by Xcode please file an issue: https://github.com/CocoaPods/Xcodeproj/issues/new

**Comment by dnkoutso** [comment]
You have a very old version of Xcodeproj gem, please update `gem update xcodeproj` and in case you are using Xcode 11.4 please lower the Xcode compatibility version in your project. I will make a release for Xcodeproj for 11.4 support maybe this week.

**Comment by valoroustech** [comment]
> You have a very old version of Xcodeproj gem, please update `gem update xcodeproj` and in case you are using Xcode 11.4 please lower the Xcode compatibility version in your project. I will make a release for Xcodeproj for 11.4 support maybe this week.
gem update xcodeproj
Updating installed gems
Nothing to update
Apples-iMac:FundSpace apple$

**Comment by RScottLewis** [comment]
I too have this issue. 
It would be great to get an update. I though i was getting this because I tried making an ios app with expo and react-native.  
This was my enviroment when I hit RuntimeError:
xcode Version 11.4 (11E146)
xcodeproj --version 1.15.0

**Comment by RScottLewis** [comment]
Here's a new twist. It didn't happen on my backup directory. This means this issue is isolated to individual project directories.

**Comment by dnkoutso** [comment]
@RScottLewis We just published 1.16.0 of Xcodeproj, please `gem update xcodeproj` which should fix this issue with Xcode 11.4.

**Comment by deliciafernandes** [comment]
> pod install
> Analyzing dependencies
> 
> ――― MARKDOWN TEMPLATE ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
> 
> ### Command
> ```
> /Applications/CocoaPods.app/Contents/Resources/bundle/bin/pod install
> ```
> 
> ### Report
> * What did you do?
> * What did you expect to happen?
> * What happened instead?
> 
> ### Stack
> ```
>    CocoaPods : 1.5.2
>         Ruby : ruby 2.2.6p396 (2016-11-15 revision 56800) [x86_64-darwin15]
>     RubyGems : 2.6.8
>         Host : Mac OS X 10.14.4 (18E226)
>        Xcode : 11.3 (11C29)
>          Git : git version 2.6.2
> Ruby lib dir : /Applications/CocoaPods.app/Contents/Resources/bundle/lib
> Repositories : master - https://github.com/CocoaPods/Specs.git @ 16f7158eb43b395cf2e77f2cb0a6141f6613806f
> ```
> 
> ### Plugins
> ```
> cocoapods-check           : 1.0.0
> cocoapods-deintegrate     : 1.0.2
> cocoapods-plugins         : 1.0.0
> cocoapods-plugins-install : 0.0.1
> cocoapods-search          : 1.0.0
> cocoapods-stats           : 1.0.0
> cocoapods-trunk           : 1.3.0
> cocoapods-try             : 1.1.0
> ```
> 
> ### Podfile
> ```ruby
> # Uncomment the next line to define a global platform for your project
>  platform :ios, '9.0'
> 
> target 'FundSpace' do
>   # Comment the next line if you're not using Swift and don't want to use dynamic frameworks
>   use_frameworks!
> 
>   # Pods for FundSpace
>   #pod 'SideMenu'
>  #pod 'PageMenu'
>  pod 'GoogleSignIn'
>  pod 'FacebookLogin'
>  pod 'Alamofire'
>  pod 'JGProgressHUD'
>  pod 'SDWebImage'
>  #pod 'SkeletonView'
>  pod 'ImageSlideshow', '~> 1.8'
>  pod 'ImageSlideshow/Alamofire'
>  pod 'AlamofireImage'
>  pod 'IQKeyboardManagerSwift'
>  pod 'SkyFloatingLabelTextField', '~> 3.0'
>  pod 'SwiftyJSON', '~> 4.0'
>  pod 'SVGKit', :git => 'https://github.com/SVGKit/SVGKit.git', :branch => '2.x'
>  pod 'ObjectMapper', '~> 3.4'
>  pod 'AlamofireObjectMapper', '~> 5.2'
>  pod 'DatePickerDialog'
>  pod 'McPicker'
>  pod 'ReachabilitySwift'
>  pod 'Firebase/Core'
>  pod 'GoogleSignIn'
>  pod 'Firebase/Database'
>  pod 'RealmSwift'
>  pod 'Firebase/Analytics'
>  pod 'AppAuth','~> 1.2.0'
>  pod 'Firebase/Auth'
>  pod 'Firebase/Storage'
>  pod 'Firebase/Messaging'
>  pod 'Firebase/RemoteConfig'
>  pod 'Charts'
>  
>  pod 'FirebaseInstanceID', "2.0.0"
>   target 'FundSpaceTests' do
>     inherit! :search_paths
>     # Pods for testing
>   end
> 
>   target 'FundSpaceUITests' do
>     inherit! :search_paths
>     # Pods for testing
>   end
> 
> end
> ```
> 
> ### Error
> ```
> RuntimeError - [Xcodeproj] Unknown object version.
> /Applications/CocoaPods.app/Contents/Resources/bundle/lib/ruby/gems/2.2.0/gems/xcodeproj-1.5.8/lib/xcodeproj/project.rb:218:in `initialize_from_file'
> /Applications/CocoaPods.app/Contents/Resources/bundle/lib/ruby/gems/2.2.0/gems/xcodeproj-1.5.8/lib/xcodeproj/project.rb:103:in `open'
> /Applications/CocoaPods.app/Contents/Resources/bundle/lib/ruby/gems/2.2.0/gems/cocoapods-1.5.2/lib/cocoapods/installer/analyzer.rb:918:in `block (2 levels) in inspect_targets_to_integrate'
> /Applications/CocoaPods.app/Contents/Resources/bundle/lib/ruby/gems/2.2.0/gems/cocoapods-1.5.2/lib/cocoapods/installer/analyzer.rb:917:in `each'
> /Applications/CocoaPods.app/Contents/Resources/bundle/lib/ruby/gems/2.2.0/gems/cocoapods-1.5.2/lib/cocoapods/installer/analyzer.rb:917:in `block in inspect_targets_to_integrate'
> /Applications/CocoaPods.app/Contents/Resources/bundle/lib/ruby/gems/2.2.0/gems/cocoapods-1.5.2/lib/cocoapods/user_interface.rb:64:in `section'
> /Applications/CocoaPods.app/Contents/Resources/bundle/lib/ruby/gems/2.2.0/gems/cocoapods-1.5.2/lib/cocoapods/installer/analyzer.rb:912:in `inspect_targets_to_integrate'
> /Applications/CocoaPods.app/Contents/Resources/bundle/lib/ruby/gems/2.2.0/gems/cocoapods-1.5.2/lib/cocoapods/installer/analyzer.rb:78:in `analyze'
> /Applications/CocoaPods.app/Contents/Resources/bundle/lib/ruby/gems/2.2.0/gems/cocoapods-1.5.2/lib/cocoapods/installer.rb:243:in `analyze'
> /Applications/CocoaPods.app/Contents/Resources/bundle/lib/ruby/gems/2.2.0/gems/cocoapods-1.5.2/lib/cocoapods/installer.rb:154:in `block in resolve_dependencies'
> /Applications/CocoaPods.app/Contents/Resources/bundle/lib/ruby/gems/2.2.0/gems/cocoapods-1.5.2/lib/cocoapods/user_interface.rb:64:in `section'
> /Applications/CocoaPods.app/Contents/Resources/bundle/lib/ruby/gems/2.2.0/gems/cocoapods-1.5.2/lib/cocoapods/installer.rb:153:in `resolve_dependencies'
> /Applications/CocoaPods.app/Contents/Resources/bundle/lib/ruby/gems/2.2.0/gems/cocoapods-1.5.2/lib/cocoapods/installer.rb:116:in `install!'
> /Applications/CocoaPods.app/Contents/Resources/bundle/lib/ruby/gems/2.2.0/gems/cocoapods-1.5.2/lib/cocoapods/command/install.rb:41:in `run'
> /Applications/CocoaPods.app/Contents/Resources/bundle/lib/ruby/gems/2.2.0/gems/claide-1.0.2/lib/claide/command.rb:334:in `run'
> /Applications/CocoaPods.app/Contents/Resources/bundle/lib/ruby/gems/2.2.0/gems/cocoapods-1.5.2/lib/cocoapods/command.rb:52:in `run'
> /Applications/CocoaPods.app/Contents/Resources/bundle/lib/ruby/gems/2.2.0/gems/cocoapods-1.5.2/bin/pod:55:in `<top (required)>'
> /Applications/CocoaPods.app/Contents/Resources/bundle/bin/pod:22:in `load'
> /Applications/CocoaPods.app/Contents/Resources/bundle/bin/pod:22:in `<main>'
> ```
> 
> ――― TEMPLATE END ――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
> 
> [!] Oh no, an error occurred.
> 
> Search for existing GitHub issues similar to yours:
> https://github.com/CocoaPods/CocoaPods/search?q=%5BXcodeproj%5D+Unknown+object+version.&type=Issues
> 
> If none exists, create a ticket, with the template displayed above, on:
> https://github.com/CocoaPods/CocoaPods/issues/new
> 
> Be sure to first read the contributing guide for details on how to properly submit a ticket:
> https://github.com/CocoaPods/CocoaPods/blob/master/CONTRIBUTING.md
> 
> Don't forget to anonymize any private data!
> 
> Looking for related issues on cocoapods/cocoapods...
> 
> * Pod init gives error message
>   #9666 [closed] [1 comment]
>   a week ago
> * ```
>      Xcode : 11.4 (11E146)
>   ```
>   
>   
>   #9662 [closed] [6 comments]
>   2 days ago
> * Xcodeproj doesn't know about the following attributes {"inputFileListPaths"=>[], "outputFileListPaths"=>[]} for the 'PBXShellScriptBuildPhase' isa
>   #7835 [closed] [54 comments]
>   a week ago
> 
> and 50 more at:
> [https://github.com/cocoapods/cocoapods/search?q=%5BXcodeproj%5D%20Unknown%20object%20version.&type=Issues&utf8=✓](https://github.com/cocoapods/cocoapods/search?q=%5BXcodeproj%5D%20Unknown%20object%20version.&type=Issues&utf8=%E2%9C%93)
> 
> [!] [!] Xcodeproj doesn't know about the following attributes {"inputFileListPaths"=>[], "outputFileListPaths"=>[]} for the 'PBXShellScriptBuildPhase' isa.
> If this attribute was generated by Xcode please file an issue: https://github.com/CocoaPods/Xcodeproj/issues/new
> 
> [!] [!] Xcodeproj doesn't know about the following attributes {"inputFileListPaths"=>["${PODS_ROOT}/Target Support Files/Pods-FundSpace/Pods-FundSpace-frameworks-${CONFIGURATION}-input-files.xcfilelist"], "outputFileListPaths"=>["${PODS_ROOT}/Target Support Files/Pods-FundSpace/Pods-FundSpace-frameworks-${CONFIGURATION}-output-files.xcfilelist"]} for the 'PBXShellScriptBuildPhase' isa.
> If this attribute was generated by Xcode please file an issue: https://github.com/CocoaPods/Xcodeproj/issues/new
> 
> [!] [!] Xcodeproj doesn't know about the following attributes {"inputFileListPaths"=>["${PODS_ROOT}/Target Support Files/Pods-FundSpace/Pods-FundSpace-resources-${CONFIGURATION}-input-files.xcfilelist"], "outputFileListPaths"=>["${PODS_ROOT}/Target Support Files/Pods-FundSpace/Pods-FundSpace-resources-${CONFIGURATION}-output-files.xcfilelist"]} for the 'PBXShellScriptBuildPhase' isa.
> If this attribute was generated by Xcode please file an issue: https://github.com/CocoaPods/Xcodeproj/issues/new
> 
> [!] [!] Xcodeproj doesn't know about the following attributes {"inputFileListPaths"=>[], "outputFileListPaths"=>[]} for the 'PBXShellScriptBuildPhase' isa.
> If this attribute was generated by Xcode please file an issue: https://github.com/CocoaPods/Xcodeproj/issues/new
> 
> [!] [!] Xcodeproj doesn't know about the following attributes {"inputFileListPaths"=>[], "outputFileListPaths"=>[]} for the 'PBXShellScriptBuildPhase' isa.
> If this attribute was generated by Xcode please file an issue: https://github.com/CocoaPods/Xcodeproj/issues/new

Can you show the` flutter doctor -v ` output? I'm facing the same thing, but due to Cocoapods issue. If nothing, run:
1. `flutter clean`
2. `pod repo update`
3. `sudo gem install cocoapods`
4. `pod setup`

If you don't get any red comments you're good to go. After this try to run you app again.
